mockserver_test.go

```diff
@@ -256,6 +257,9 @@ func (s *server) handle(conn net.Conn) {
 		l := sc.Text()
 		parts := strings.Split(l, " ")
 		cmd := strings.TrimSpace(parts[0])
+		if cmd == "clientlist" {
```

Comment:
A little comment here wouldn't go a miss as it's clear from first read why this is needed.

Reply:
The new commit holds a comment where I try to explain why this was necessary for testing with options purposes.